packages/interaction-context/src/InteractionContext.tsx

```diff
@@ -0,0 +1,54 @@
+import React, { createContext, useState } from 'react';
+import PropTypes from 'prop-types';
+import { useEventListener } from '@leafygreen-ui/hooks';
+
+export const UsingKeyboardContext = createContext(undefined as
+  | boolean
+  | undefined);
+
+interface InteractionContextProps {
+  children: React.ReactNode;
+  initialStates?: {
```

Comment:
nit

```suggestion
  initialState?: {
```

nit

Reply:
There were more places to change this, so I did so manually. Either way, done.